docs/changelog.md

```diff
@@ -151,18 +171,20 @@ description: |
 
 - Introspection
   - Implemented a new system by which Typst tracks where elements end up on the
-    pages. This may lead to subtly different behaviour in introspections.
+    pages. This may lead to subtly different behavior in introspections.
     (**Breaking change**)
-  - Fixed various bugs with wrong counter behaviour in complex layout
+  - Fixed various bugs with wrong counter behavior in complex layout
     situations, through a new, more principled implementation
   - Counter updates can now be before the first, in between, and after the last
     page when isolated by weak page breaks. This allows, for instance, updating
     a counter before the first page header and background.
+    <!-- Counter updates can now be in between pages?
```

Comment:
No. Normal counter updates are just somewhere on the page. what this means is that this
```typ
#set page(numbering: "1", number-align: top)
#counter(page).update(5)
```
is different from
```typ
#counter(page).update(5)
#set page(numbering: "1", number-align: top)
```
because the counter is before the first page (the set page creates a weak pagebreak).

This is a new capability that didn't exist before.

Reply:
But you describe the "before the first" part, which I already more or less understood. Do you mean that the changelog is wrong and "in between" must be removed? I don't understand.

Reply:
In between means
```typ
#pagebreak(weak: true)
#counter(page).update(5)
#pagebreak(weak: true)
```
which is logically between the first page's footer and second page's header